app/assets/stylesheets/base.scss

```diff
@@ -137,6 +137,15 @@ textarea {
   }
 }
 
+.truncate-ellipsis .status-label {
+    display: table-cell;
+}
+
```

Comment:
@Rynaro [This](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#mergeableselector) might help.